src/player.cpp

```diff
@@ -2097,11 +2097,36 @@ Item* Player::getCorpse(Creature* lastHitCreature, Creature* mostDamageCreature)
 {
 	Item* corpse = Creature::getCorpse(lastHitCreature, mostDamageCreature);
 	if (corpse && corpse->getContainer()) {
+		const auto& killers = getKillers();
+		std::map<std::string, uint16_t> names;
+		for (auto& killer : killers) {
+			const std::string& killerName = killer->getName();
+			if (names.find(killerName) != names.end()) {
+				names[killerName]++;
```

Comment:
can you explain what's the point of this code?

Reply:
There is nothing to explain, I usually get a little confused and go blind, thank you for making me see these things 😍, thank God that there are people like you to review things thoroughly